controller_interval: 0.1

bot_orientation_messages_maxlength: 10

shutdown_coords_global: [10.981856913385355, 20.638044400288646]
shutdown_tolerance_distance: 9
resume_coords_global: [-14.127625860234343, 21.478713612828365]
resume_tolerance_distance: 1

start_goal_distance: 5
resume_goal_distance: 5

goal_angle_threshold: 91
goal_distance_max: 14
goal_distance_min: 1.5

dbscan_eps: 12
dbscan_min_samples: 4

midpoint_tolerance: 0.1

minimum_points: 200